src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+import seedu.address.model.meeting.Meeting;
+
+/**
+ * Deletes a Meeting of specified client identified using it's displayed index from MyInsuRec.
+ */
+public class DeleteMeetingCommand extends Command {
+    public static final String COMMAND_WORD = "delMeeting";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the meeting of the client "
+            + "by the index number of the client. \n"
+            + "Parameters: INDEX (must be a positive integer) \n"
+            + "Example: " + COMMAND_WORD + " i/1";
+
+    public static final String MESSAGE_ADD_DELETE_MEETING_SUCCESS = "Remove meeting from Client: %1$s";
+    public static final String MESSAGE_MEETING_NOT_FOUND = "Meeting not found from Client: %1$s";
+
+    private final Index index;
+
+    /**
+     * Creates an DeleteMeetingCommand to delete the specified client's meeting
+     */
+    public DeleteMeetingCommand(Index index) {
+        requireAllNonNull(index);
+
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        List<Client> lastShownList = model.getFilteredClientList();
+
+        if (index.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_MEETING_DISPLAYED_INDEX);
+        }
+
+        Client clientToEdit = lastShownList.get(index.getZeroBased());
+        Client editedClient = new Client(
+                clientToEdit.getName(), clientToEdit.getPhone(), clientToEdit.getEmail(),
+                clientToEdit.getAddress(), clientToEdit.getTags()
+        );
+        Meeting meeting = clientToEdit.getMeeting();
+
+        if (meeting == null) {
+            throw new CommandException(String.format(MESSAGE_MEETING_NOT_FOUND, clientToEdit));
+        }
+
+        model.deleteMeeting(meeting);
+        model.setClient(clientToEdit, editedClient);
```

Comment:
This method seems to remove all meetings from client rather than just 1 meeting from the client, since `editedClient` has an empty field. Possible problem in the future since this means that DeleteMeeting deletes all the meetings rather than the selected meeting.

Reply:
However, there is no other easy way to do this for now. Furthermore, each Client only has one Meeting for now (so this method isn't wrong). In the future when we transit to using List<Meeting>, this will require changes.

Reply:
Yup, will take note of this when we start implementing meeting as a list in client. Now I just do it such that it can work for our current iteration